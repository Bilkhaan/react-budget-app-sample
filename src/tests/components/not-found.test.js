import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../components/not-found';

test('Should render NotFound component correctly', () => {
  const wrapper = shallow(<NotFound />);
  expect(wrapper).toMatchSnapshot();
});