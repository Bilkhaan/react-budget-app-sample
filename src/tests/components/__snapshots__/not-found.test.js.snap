// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render NotFound component correctly 1`] = `
<div>
  404!
</div>
`;
